doc/examples/xx_applications/plot_morphology.py

```diff
@@ -240,3 +240,5 @@ def plot_comparison(original, filtered, filter_name):
 # /ImageProcessing-html/topic4.htm>`_
 #
 # 3. http://en.wikipedia.org/wiki/Mathematical_morphology
+
+plt.show()
```

Comment:
I think this should belong to `plot_comparison` function.

Reply:
The options are to show the figures successively or at once at the end of the script. I prefer the latter but it's the choice of the maintainers...